python/heterocl/devices.py

```diff
@@ -242,23 +250,23 @@ class env(type):
     def __getattr__(cls, key):
```

Comment:
This class is inherited from `type` instead of `object`. the `__getattr__` is a metaclass method, so we should use cls here: https://stackoverflow.com/questions/37826361/how-to-understand-this-example-of-getattr-as-metaclass-method